libs/ui-watt/src/lib/foundations/typography/_typography.import.scss

```diff
@@ -62,7 +62,7 @@ $font-sizes: (
 }
 
 @mixin watt-headline-1 {
-  color: var(--watt-typography-headline-color);
+  color: var(--watt-typography-text-color);
```

Comment:
question: why not change the color of `--watt-typography-headline-color` instead?

Reply:
That would be better

Reply:
Resolved in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1497/commits/5e8c97e4f562ff9bdd8db4384141da5da112bb4b